### PR TITLE
throw friendly errors in /user/add

### PR DESCRIPTION
**Pull request by groovecoder**
I ran OWASP ZAP at my localhost and also found another error trace when email was left off.

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const TIPS = require("../tips");
 async function add(req, res) {
   const email = req.body.email;
 
-  if (!isemail.validate(email)) {
+  if (!email || !isemail.validate(email)) {
```
Comment (pdehaan) [inline controllers/user.js]: Me thinks this works!

```
$ curl -XPOST http://localhost:6060/user/add | grep -A1 "<h2>Error</h2>"

    <h2>Error</h2>
    <p> Invalid Email </p>
```

File: controllers/user.js [hunk 2]
```diff
@@ -37,6 +37,9 @@ async function add(req, res) {
 
 
 async function verify(req, res) {
+  if (!req.query.token) {
+    throw new Error("Verification token is required.");
+  }
```
Comment (pdehaan) [inline controllers/user.js]: 👍 

```sh
$ curl http://localhost:6060/user/verify | grep -A1 "<h2>Error</h2>"

    <h2>Error</h2>
    <p> Verification token is required. </p>
```

Comment (pdehaan) [reply controllers/user.js]: The error messages are friendlier, but not sure if we want to add UUID validation or anything, or if that is for a future PR...

Like, if I specify a bogus/typoed `?token=3` query param, it seems to do a DB query on the invalid UUID.v4.

```bash
$ curl https://fx-breach-alerts.herokuapp.com/user/verify\?token\=3 | grep -A1 "<h2>Error</h2>"

    <h2>Error</h2>
    <p> This email address is not subscribed to Firefox Monitor. </p>
```

**Comment by groovecoder** [comment]
Merging this as it fixes OWASP scan alerts. Will leave #255 open.

**Pull request merged**